sematic/ui/packages/common/src/component/GitInfo.tsx

```diff
@@ -25,30 +29,67 @@ const StyledBox = styled(Box)`
 `;
 
 interface GitInfoBoxProps {
+    hasGitInfo?: boolean;
     hasUncommittedChanges?: boolean;
+    commit?: string;
+    branch?: string;
+    commitLink?: string;
+    branchLink?: string;
 }
 
-const GitInfoBox = (prop: GitInfoBoxProps) => {
-    const { hasUncommittedChanges = false } = prop;
+export const GitInfoBoxPresentation = (prop: GitInfoBoxProps) => {
+    const { hasUncommittedChanges = false, hasGitInfo = true, branch, branchLink,
+        commit, commitLink } = prop;
+
+    if (!hasGitInfo) {
+        return <StyledBox><Typography color="GrayText" variant={"small"}>
+            No Git information.
+        </Typography></StyledBox>
+    }
 
     return <>
         <StyledBox>
             <RiGitBranchLine />
-            <Link variant={"small"}>acme/feature-branch</Link>
-            <CopyButton text={"acme/feature-branch"} />
+            <Link variant={"small"} href={branchLink} target="_blank">{branch}</Link>
+            <CopyButton text={branch!} />
```

Comment:
Clicking these copy buttons no longer produces a toast saying `"Copied."`. Is this intentional?

Reply:
Not intentional. It was because the `<SnackbarProvider>` hadn't been adequately implemented. In the new iteration, I implemented it, including some cleanup and simplification. (Turns out some features are not actually being used)